src/test/java/org/broadinstitute/hellbender/tools/walkers/rnaseq/ASEReadCounterIntegrationTest.java

```diff
@@ -1,121 +1,75 @@
-* copy, modify, merge, publish, distribute, sublicense, and/or sell
-* copies of the Software, and to permit persons to whom the
-* Software is furnished to do so, subject to the following
-* conditions:
-* 
-* The above copyright notice and this permission notice shall be
-* included in all copies or substantial portions of the Software.
-* 
-* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
-* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
-* OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
-* NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
-* HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
-* WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
-* FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR
-* THE USE OR OTHER DEALINGS IN THE SOFTWARE.
-*/
+package org.broadinstitute.hellbender.tools.walkers.rnaseq;
 
-package org.broadinstitute.gatk.tools.walkers.rnaseq;
-
-import org.broadinstitute.gatk.engine.walkers.WalkerTest;
-import org.broadinstitute.gatk.utils.exceptions.UserException;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.utils.test.IntegrationTestSpec;
+import org.broadinstitute.hellbender.exceptions.UserException;
 import org.testng.annotations.Test;
 
 import java.util.Arrays;
 
-/**
- * Test all different parameters.
- */
-public class ASEReadCounterIntegrationTest extends WalkerTest {
-
-    @Test
-    public void testASEReadCounterWithHighMQ() {
-        WalkerTest.WalkerTestSpec spec = new WalkerTest.WalkerTestSpec(
-                "-T ASEReadCounter -R " + b37KGReference + " -I " + privateTestDir + "NA12878.RNAseq.bam -sites "+privateTestDir +"NA12878.chr20_2444518_2637800.RNAseq.SYNONYMOUS_CODING.vcf -mmq 60 -o %s -U ALLOW_N_CIGAR_READS", 1,
-                Arrays.asList("eec421405a4a570751821d158734020e"));
-        executeTest("test high mq with no read passing", spec);
-    }
+public final class ASEReadCounterIntegrationTest extends CommandLineProgramTest {
 
     @Test
-    public void testASEReadCounterWithLowMQ() {
-        WalkerTest.WalkerTestSpec spec = new WalkerTest.WalkerTestSpec(
-                "-T ASEReadCounter -R " + b37KGReference + " -I " + privateTestDir + "NA12878.RNAseq.bam -sites "+privateTestDir +"NA12878.chr20_2444518_2637800.RNAseq.SYNONYMOUS_CODING.vcf -mmq 1 -o %s -U ALLOW_N_CIGAR_READS", 1,
-                Arrays.asList("4f1144be0bb2c4adeae00625afd04cda"));
-        executeTest("test high mq with no read passing", spec);
+    public void testASEReadCounterWithHighMQ() throws Exception {
+        IntegrationTestSpec spec = new IntegrationTestSpec(
+                "-R " + b37_reference_20_21 + " -I " + largeFileTestDir + "NA12878.RNAseq.bam -V " + publicTestDir + "NA12878.chr20_2444518_2637800.RNAseq.SYNONYMOUS_CODING.vcf -O %s -mmq 60 ",
+                Arrays.asList(publicTestDir + "expected.ASEReadCount.WithHighMQ.table"));
```

Comment:
Could you move the test files into `src/test/resources/org/broadinstitute/hellbender/tools/walkers/rnaseq/ASEReadCounter/`? 

It keep things a bit cleaner and easier to understand what a file is used for.